angular/src/lib/sidebar-nav-item/sidebar-nav-item.component.ts

```diff
@@ -50,18 +51,38 @@ import { SidebarService } from '../sidebar/sidebar.service';
       <!-- Icon conditional here -->
       <div class="md-list-item__left" *ngIf="icon">
         <i class="md-icon icon" [ngClass]="[
-          icon ? 'icon-' + this.icon : ''
-        ]" style="font-size: 20px;"></i>
+          icon ? 'icon-' + this.icon : '',
+          iconClass
+        ]"></i>
       </div>
 
-      <div class="md-list-item__center">{{ title }}</div>
+      <ng-container *ngIf="avatarTitle && !icon">
+        <div class="md-list-item__left">
+          <md-avatar
```

Comment:
@pauljeter daani and I looked at this yesterday and didn't know if the footer avatar should be considered as a sidebar-nav-item

Reply:
Avatar taken out of sidebar code.